### DISCUSSION TITLE
Neotree update nvim cwd

### DISCUSSION THREAD

**Discussion by NielsenEDI** [LazyVim/LazyVim#1398] [Q&A]
Is there a way for me to make it so that when i change neotree working directory with . so that the current directory gets updated correctly in nvim? or is it clever to not have it change?
I dont understand this functionality.

Thanks in advance, its one of the last things thats preventing me from switching to lazyvim from vscode.

**Comment by dpetka2001** [comment]
Neo-tree has `bind_to_cwd` which in Lazyvim is set to false by default. I believe if you change that to true, it will behave like you want.
